internal/fs/hns_bucket_test.go

```diff
@@ -48,8 +48,13 @@ var expectedFooDirEntries = []dirEntry{
 func TestHNSBucketTests(t *testing.T) { suite.Run(t, new(HNSBucketTests)) }
 
 func (t *HNSBucketTests) SetupSuite() {
-	t.serverCfg.NewConfig = &cfg.Config{EnableHns: true}
 	t.serverCfg.ImplicitDirectories = false
+	t.serverCfg.NewConfig = &cfg.Config{
+		Write: cfg.WriteConfig{
+			CreateEmptyFile: false,
```

Comment:
isn;t this false by default?

Reply:
I thought showing it explicitly makes code more readable.